manifests/c/CodecGuide/K-LiteCodecPack/Standard/17.3.0/CodecGuide.K-LiteCodecPack.Standard.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: CodecGuide.K-LiteCodecPack.Standard
+PackageVersion: 17.3.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://files3.codecguide.com/K-Lite_Codec_Pack_1730_Standard.exe
+  InstallerSha256: 2AC426594EF3ADA86EE2ED49B1756C28EB86E2054EAD064A49571A85D922B537
+  AppsAndFeaturesEntries:
+  - DisplayName: K-Lite Codec Pack 17.2.5 Standard

```

```suggestion
  - DisplayName: K-Lite Codec Pack 17.3.0 Standard
```
This must match Control Panel.